nixos/modules/programs/clash-verge.nix

```diff
@@ -1,9 +1,17 @@
-{ config, lib, pkgs, ... }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
 
 {
   options.programs.clash-verge = {
     enable = lib.mkEnableOption "Clash Verge";
-    package = lib.mkPackageOption pkgs "clash-verge" {};
+    package = lib.mkPackageOption pkgs "Clash Verge" {
+      default = "clash-verge-rev";
```

Comment:
Oh, here is one: we are letting people switch to clash-verge-rev without telling them this clearly.

Reply:
This would be better to use

```nix
lib.mkOption {
  type = lib.types.package;
  description = ''
    The clash-verge package to use. Available options are
    clash-verge-rev and clash-nyanpasu, both are forks of
    the original clash-verge project.
  '';
  example = "pkgs.clash-verge-rev";
}
```

Without setting the default value.